packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.8.4
+
+* Update example to call `super.dispose()` last
```

Comment:
Updates

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style